### PR TITLE
fix c8 api sidebar

### PR DESCRIPTION
**Pull request by christinaausley**
## Description

Minor sidebar fix for C8 API.

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [ ] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [x] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [ ] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [x] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

**Review by conceptualshark — APPROVED** [review]

File: docs/apis-tools/camunda-api-rest/specifications/sidebar.js [hunk 1]
```diff
@@ -163,12 +163,6 @@ module.exports = [
         label: "Modify process instance",
         className: "api-method post",
       },
-    ],
-  },
-  {
-    type: "category",
-    label: "Process Instance",
```
Comment (tmetzke) [inline docs/apis-tools/camunda-api-rest/specifications/sidebar.js:170]: @christinaausley, I guess this reflects an inconsistent capitalization in our tags in the OpenAPI spec. This is why we have two different categories here.

Comment (christinaausley) [reply docs/apis-tools/camunda-api-rest/specifications/sidebar.js:170]: @tmetzke Can you remind me the link to the spec, and I can launch a PR for that?

Comment (tmetzke) [reply docs/apis-tools/camunda-api-rest/specifications/sidebar.js:170]: Here you go: https://github.com/camunda/camunda/blob/main/zeebe/gateway-protocol/src/main/proto/rest-api.yaml

Comment (tmetzke) [reply docs/apis-tools/camunda-api-rest/specifications/sidebar.js:170]: I have fixed this over here: https://github.com/camunda/camunda/pull/22755
Let me know if you have fixed this in a PR as well already, I couldn't find any fix for this yet 🙈

Comment (christinaausley) [reply docs/apis-tools/camunda-api-rest/specifications/sidebar.js:170]: Done with https://github.com/camunda/camunda/pull/22942/.

Comment (christinaausley) [reply docs/apis-tools/camunda-api-rest/specifications/sidebar.js:170]: This PR resolves this duplicate, and everything is now listed as items under Process instance. Were you expecting additional changes?

Comment (tmetzke) [reply docs/apis-tools/camunda-api-rest/specifications/sidebar.js:170]: No, sorry, I was just referring to the fix in the source spec and was wondering if you had created a PR for the source spec as well. I couldn't find one so I fixed this in https://github.com/camunda/camunda/pull/22755. Apparently, you had a PR open already for that which became a duplicate then, sorry for that 🙈

**Pull request merged**